bundles/org.openhab.binding.carnet/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="bridge-type:carnet:account">
+		<parameter-group name="authentication">
+			<label>Authentication</label>
+			<description>Authentication settings.</description>
+		</parameter-group>
+
+		<parameter name="brand" type="text" required="true">
+			<label>Brand</label>
+			<description>CarNet Brand.</description>
+			<options>
+				<option value="Audi">Audi</option>
+			</options>
+			<default>Audi</default>
+		</parameter>
+		<parameter name="country" type="text" required="true">
+			<label>Market (Country)</label>
+			<description>Market in which is associated (must not match the country where the vehicle was bought).</description>
+			<options>
+				<option value="DE">Europe</option>
+				<option value="United States">US</option>
+				<option value="China">CN</option>
+			</options>
+			<default>DE</default>
+		</parameter>
+		<parameter name="user" type="text" required="true" groupName="authentication">
+			<label>Online User</label>
+			<description>User name for the CarNet Login.</description>
+		</parameter>
+		<parameter name="password" type="text" required="true" groupName="authentication">
+			<label>Password</label>
+			<context>password</context>
+			<description>Password to access the CarNet Account.</description>
+		</parameter>
+
+		<parameter name="pollingInterval" type="integer" required="false" min="5">
+			<label>Polling Interval</label>
+			<description>Status update interval in minutes.</description>
+			<default>15</default>
+		</parameter>
+	</config-description>
+
+	<config-description uri="thing-type:carnet:vehicle">
+		<parameter-group name="advancedOptions">
+			<label>Advanced Options</label>
+		</parameter-group>
+
+		<parameter name="pin" type="text" required="false" groupName="authentication">
+			<label>PIN</label>
+			<context>password</context>
+			<description>Security PIN to access control functions.</description>
+		</parameter>
+
+		<parameter name="numShortTrip" type="integer" required="false" min="0" max="5">
+			<label>Entries in Short Trip history</label>
```

Comment:
Can you make this more concise? Guideline is 2-3 words with up to 25 chars.

Also words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Same for the others.

Reply:
what's about "Short-Trip History Size"?

Reply:
Sounds good.

Reply:
done